gems/pending/spec/spec_helper.rb

```diff
@@ -12,6 +14,8 @@
 # Initialize the global logger that might be expected
 require 'logger'
 $log ||= Logger.new("/dev/null")
+# $log ||= Logger.new(STDOUT)
+# $log.level = Logger::DEBUG
```

Comment:
I wasn't sure if it's acceptable to use `puts` in these scripts to output information directly. Displaying the meaning of environment variables and their current settings would be useful.